specification/cognitiveservices/data-plane/AnomalyDetector/stable/v1.1/cadl/univariate/models.cadl

```diff
@@ -0,0 +1,304 @@
+  maxAnomalyRatio?: float32;
+
+  @doc("""
+Optional argument, advanced model parameter, between 0-99, the lower the value
+is, the larger the margin value will be which means less anomalies will be
+accepted.
+""")
+  sensitivity?: int32;
+
+  @doc("""
+Used to specify how to deal with missing values in the input series, it's used
+when granularity is not \"none\".
+""")
+  imputeMode?: ImputeMode;
+
+  @doc("""
+Used to specify the value to fill, it's used when granularity is not \"none\"
+and imputeMode is \"fixed\".
+""")
+  imputeFixedValue?: float32;
+}
+
+@doc("The definition of input timeseries points.")
+model TimeSeriesPoint {
+  @doc("Optional argument, timestamp of a data point (ISO8601 format).")
+  timestamp?: zonedDateTime;
+
+  @doc("The measurement of that point, should be float.")
+  value: float32;
+}
+
+@doc("The response of entire anomaly detection.")
+model UnivariateDetectionResult {
+  @doc("""
+Frequency extracted from the series, zero means no recurrent pattern has been
+found.
+""")
+  period: int32;
+
+  @doc("""
+ExpectedValues contain expected value for each input point. The index of the
+array is consistent with the input series.
+""")
+  expectedValues: float32[];
+
+  @doc("""
+UpperMargins contain upper margin of each input point. UpperMargin is used to
+calculate upperBoundary, which equals to expectedValue + (100 -
+marginScale)*upperMargin. Anomalies in response can be filtered by
+upperBoundary and lowerBoundary. By adjusting marginScale value, less
+significant anomalies can be filtered in client side. The index of the array is
+consistent with the input series.
+""")
+  upperMargins: float32[];
+
+  @doc("""
+LowerMargins contain lower margin of each input point. LowerMargin is used to
+calculate lowerBoundary, which equals to expectedValue - (100 -
+marginScale)*lowerMargin. Points between the boundary can be marked as normal
+ones in client side. The index of the array is consistent with the input
+series.
+""")
+  lowerMargins: float32[];
+
+  @doc("""
+IsAnomaly contains anomaly properties for each input point. True means an
+anomaly either negative or positive has been detected. The index of the array
+is consistent with the input series.
+""")
+  isAnomaly: boolean[];
+
+  @doc("""
+IsNegativeAnomaly contains anomaly status in negative direction for each input
+point. True means a negative anomaly has been detected. A negative anomaly
+means the point is detected as an anomaly and its real value is smaller than
+the expected one. The index of the array is consistent with the input series.
+""")
+  isNegativeAnomaly: boolean[];
+
+  @doc("""
+IsPositiveAnomaly contain anomaly status in positive direction for each input
+point. True means a positive anomaly has been detected. A positive anomaly
+means the point is detected as an anomaly and its real value is larger than the
+expected one. The index of the array is consistent with the input series.
+""")
+  isPositiveAnomaly: boolean[];
```

Comment:
Java apiview: these "is###" with type as list could be strange to some SDK.

E.g. java
```java
        public List<Boolean> getIsAnomaly() 
        public List<Boolean> getIsNegativeAnomaly() 
        public List<Boolean> getIsPositiveAnomaly() 
```

What about other SDKs?